apps/console/src/features/console-settings/components/console-administrators/invite-new-administrator-wizard/invite-new-administrator-wizard.tsx

```diff
@@ -172,6 +193,9 @@ const InviteNewAdministratorWizard: FunctionComponent<InviteNewAdministratorWiza
                         )
                     });
                 }
+            })
+            .finally(() => {
+                onClose(null, null);
```

Comment:
Since we show the result of the invite process as alerts, the user will not be able to see the error messages if we don't close the wizard (Because of the modal dimmer).

Reply:
Fixed with 03ba80111df5014d2dad03169fadff38f9f650eb